sandbox-tools/merge-repos/src/syncReposToStaging.ts

```diff
@@ -362,22 +317,38 @@ async function resolveConflictsToTheirs(git: SimpleGit, commitDetails: ICommitDe
         let conflicted = status.conflicted[lp];
         let fileStatus = getFileStatus(status, conflicted);
         let fileState = `${fileStatus.index.padEnd(1)}${fileStatus.working_dir.padEnd(1)}`.replace(/\s/g, "_");
+
         commitSummary[fileState] = (commitSummary[fileState] + 1 || 1);
+
         if (fileStatus.index === "D") {
             // index      workingDir     Meaning
             // -------------------------------------------------
             // D                  deleted from index
             // D             D    unmerged, both deleted
             // D             U    unmerged, deleted by us
-            commitMessage = logAppendMessage(commitMessage, fileStatus, "Removed from theirs");
-            await git.rm(conflicted);
+            if (fileStatus.working_dir === "U") {
+                // Deleted by Us
+                commitMessage = logAppendMessage(commitMessage, fileStatus, "Deleted by us => checkout theirs");
+                await git.checkout([
+                    "--theirs",
+                    conflicted
+                ]);
+
+                await git.raw([
```

Comment:
Changed the basic `git.add <file>` to `git add -f <file>` as the CONTRIB repo had some files (which ended up having merge conflicts) that are explicitly excluded via .gitignore -- so needed to force the add.